CMakeLists.txt

```diff
@@ -539,6 +539,18 @@ if (LBANN_WITH_PYTHON_FRONTEND OR LBANN_WITH_EMBEDDED_PYTHON)
 
 endif (LBANN_WITH_PYTHON_FRONTEND OR LBANN_WITH_EMBEDDED_PYTHON)
 
+if (LBANN_WITH_PYTHON_FRONTEND)
+  set(LBANN_PFE_PYTHON_EXECUTABLE "${Python_EXECUTABLE}" CACHE FILEPATH "")
+  set(LBANN_PFE_PYTHONPATH "" CACHE STRING "")
+  configure_file(
+    "${CMAKE_SOURCE_DIR}/cmake/configure_files/lbann_pfe.sh.in"
+    "${CMAKE_BINARY_DIR}/lbann_pfe.sh"
```

Comment:
I wonder if it should be a `.sh` file or just an extensionless file with a `#!` header